### ISSUE TITLE
Cargo check gives error  no `WasmClientBuilder` in the root in wasm

### ISSUE THREAD

**Issue by amiyatulu** [paritytech/jsonrpsee#965]
While using jsonrpsee_wasm_client in wasm/yew, cargo check gives error:
when you use:
use jsonrpsee_wasm_client::WasmClientBuilder;

[dependencies]
jsonrpsee-wasm-client = "0.16.1"
jsonrpsee-core = "0.16.1"

But the code gets successfully compiled.

error[E0432]: unresolved import `jsonrpsee_wasm_client::WasmClientBuilder`
 --> src/components/pages/rpc_call.rs:8:5
  |
8 | use jsonrpsee_wasm_client::WasmClientBuilder;
  |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ no `WasmClientBuilder` in the root

**Comment by niklasad1** [comment]
Yeah, I believe this is because we have some conditional compilation based on `#[cfg(tests/tests/wasm.rs:#![cfg(target_arch = "wasm32")]`

Should be possible to relax that for sure.

**Comment by niklasad1** [comment]
If you change `target = wasm32-unknown-unknown` in your project it should go away

**Comment by amiyatulu** [comment]
It doesn't go away with 
`rustup target add wasm32-unknown-unknown`
